web/app/(home)/MetalakesView.js

```diff
@@ -15,8 +15,8 @@ const MetalakesView = () => {
       <Grid item xs={12}>
         <Typography className={'twc-mb-4 twc-text-[1.375rem] twc-font-bold'}>Metalakes</Typography>
         <Typography sx={{ color: 'text.secondary' }}>
-          A metalake is the top-level container for data in Gravitino. With a metalake, Gravitino provides a 3-level
-          namespace for organizing data: catalog, schemas (also called databases), and tables / views.
+          A metalake is the top-level container for data in Gravitino. Within a metalake, Gravitino provides a 3-level
+          namespace for organizing data: catalog, schemas /databases, and tables / views.
```

Comment:
Can you please add whitespace before "databases", like `/ databases`?

Reply:
correct usage to to have no spaces

Reply:
https://www.grammarly.com/blog/slash/